pkg/permissionclaim/permissionclaim_labeler.go

```diff
@@ -148,3 +144,13 @@ func (l *Labeler) LabelsFor(ctx context.Context, cluster logicalcluster.Name, gr
 
 	return labels, nil
 }
+
+/*
+installIndexers method is added, as we are re-electing the controllers
+as the informers are always running, we cannot call this in multiple places.
+*/
```

Comment:
Nit: These comments are formatted in a very atypical way for Go. Can you please update them (here and in the other occurrences of `InstallIIndexers`) to have `//` in front of each line and follow the style of _"FunctionName does this or that"_, so it would be something like

> InstallIndexers adds the additional indexers that this controller requires to the informers.

Reply:
Done